dgr/common/aci-manifest.go

```diff
@@ -96,20 +97,38 @@ func WriteAciManifest(m *AciManifest, targetFile string, projectName string, dgr
 
 	//dgrBuilderIdentifier, _ := types.NewACIdentifier(ManifestDrgBuilder)
 	dgrVersionIdentifier, _ := types.NewACIdentifier(ManifestDrgVersion)
-	buildDateIdentifier, _ := types.NewACIdentifier("build-date")
 	im.Annotations.Set(*dgrVersionIdentifier, dgrVersion)
-	//im.Annotations.Set(*dgrBuilderIdentifier, m.Builder.Image.String())
-	im.Annotations.Set(*buildDateIdentifier, time.Now().Format(time.RFC3339))
+
+	if _, ok := im.Annotations.Get("build-date"); !ok {
+		buildDateIdentifier, _ := types.NewACIdentifier("build-date")
+		im.Annotations.Set(*buildDateIdentifier, time.Now().Format(time.RFC3339))
+	}
+
 	im.Dependencies, err = ToAppcDependencies(m.Aci.Dependencies)
 	if err != nil {
 		return errs.WithEF(err, fields, "Failed to prepare dependencies for manifest")
 	}
 	im.Name = *name
 	im.Labels = labels
 
+	for _, exclusion := range m.Build.Exclude {
+		if strings.HasPrefix("/dgr/bin", exclusion) {
+			goto collectionIsComplete
```

Comment:
After reading 10 times, it start to see you want to exclude completly dgr prestart process if /dgr/bin is excluded.

Is it needed for reproductible build or can you add it in another PR ?